includes/wikia/nirvana/WikiaApp.class.php

```diff
@@ -433,20 +435,26 @@ public function registerApiController( $className, $path ) {
 	 *
 	 * onArticleViewHeader
 	 *
-	 * This is a hook which serves the needs of registerNamespaceControler
+	 * This is a hook which serves the needs of registerNamespaceController
 	 *
 	 * @param Article $article
 	 * @param bool $outputDone
 	 * @param bool $useParserCache
 	 * @return bool
 	 */
 
-	public function onArticleViewHeader(&$article, &$outputDone, &$useParserCache) {
+	public function onArticleViewHeader( &$article, &$outputDone, &$useParserCache ) {
 		$title = $article->getTitle();
 
 		$namespace = $title->getNamespace();
-		if( !empty($this->namespaceRegistry[$namespace]) && (empty($this->namespaceRegistry['exists']) || $title->exists()) ) {
-			$this->wg->Out->addHTML($this->renderView($this->namespaceRegistry[$namespace]['className'], $this->namespaceRegistry[$namespace]['methodName'], array( 'title' => $article->getTitle() ) ));
```

Comment:
The check for `empty( $this->namespaceRegistry['exists'] )` is always `true` since the exists value is actually set on `$this->namespaceRegistry[$namespace]['exists']`.  Removed this useless check.